src/components/sidebar/sidebar-component.jsx

```diff
@@ -8,11 +8,11 @@ import FixedHeader from 'components/fixed-header';
 import uiStyles from 'styles/ui.module';
 import styles from './sidebar-styles.module.scss';
 
-const Sidebar = ({ map, theme, children, activeCategory, handleSidebarToggle, isSidebarOpen, isLandscapeMode }) => {
+const Sidebar = ({ map, theme, children, activeCategory, handleSidebarToggle, isSidebarOpen, isLandscapeMode, isFullscreenActive }) => {
   const slide = useSpring({
     from: { marginLeft: -400 },
-    marginLeft: isSidebarOpen && !isLandscapeMode ? 0 : -400,
-    delay: isSidebarOpen && !isLandscapeMode ? 400 : 0
+    marginLeft: isSidebarOpen && !isLandscapeMode && !isFullscreenActive ? 0 : -400,
```

Comment:
I think we can think about how to simplify this kind of condition chain, after the first release ofc :)